repository/user_repository.go

```diff
@@ -11,6 +11,7 @@ import (
 
 const (
 	getUserByEmailQuery = "SELECT id, name, email, password, is_admin FROM users WHERE email= $1"
+	selectAllUsers      = "select id,name,email,password,is_admin,created_at,updated_at from users"
```

Comment:
lets follow formats where we keep the keywords in CAPS as SELECT, FROM, WHERE etc to be more readable.